### PR TITLE
Convert the API code base to use single quotes consistently.

### PR DESCRIPTION
**Pull request by jguertin**
Includes the addition of `flake8-quotes` extension for flake8.

### CODE REVIEW

File: .github/workflows/compile-and-test-stage.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
         run: |
           pylint -E src
           pylint -E --disable=E1101 migrations
-          flake8 src migrations tests
+          flake8 src migrations tests *.py
```
Comment (jguertin) [inline .github/workflows/compile-and-test-stage.yml:34]: Just adding python files in the PPR directory to this linting

File: ppr-api/requirements/dev.txt [hunk 2]
```diff
@@ -4,6 +4,7 @@
 autopep8
 fastapi==0.42.0 # version tied to https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker
 flake8==3.7.9
+flake8-quotes==2.1.1
```
Comment (jguertin) [inline ppr-api/requirements/dev.txt:7]: What do you think, should we keep this? https://pypi.org/project/flake8-quotes/2.1.1/ https://github.com/zheller/flake8-quotes

By default it enforces single-quotes, and as it extends flake8, no additional configuration is needed unless we move away from the defaults.  That said, I'm not clear how safe the source is, nor how widely used this is.

It does make enforcement easy.

@WalterMoar @gh20s

Comment (WalterMoar) [reply ppr-api/requirements/dev.txt:7]: I like easy enforcement, and I'm jealous :)

**Review by WalterMoar — APPROVED** [review]

**Review by gh2os — APPROVED** [review]

File: ppr-api/requirements/dev.txt (hunk 2 above)
Comment (gh2os) [reply ppr-api/requirements/dev.txt:7]: I like it as well :)

**Pull request merged**